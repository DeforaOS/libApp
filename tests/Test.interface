#$Id$
service=Test

[call::Test]
ret=VOID
arg1=INT32,i32

[call::Test2]
ret=BOOL
arg1=INT32_INOUT,i32

[call::Test3]
ret=STRING
arg1=INT32_INOUT
