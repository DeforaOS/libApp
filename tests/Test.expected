/* $Id$ */



#ifndef Test_Test_H
# define Test_Test_H

# include <stdbool.h>
# include <stdint.h>
# include <System/App.h>


/* calls */
void Test_Test(App * app, AppServerClient * client, int32_t i32);
bool Test_Test2(App * app, AppServerClient * client, int32_t * i32);
String const * Test_Test3(App * app, AppServerClient * client, int32_t *);

#endif /* !Test_Test_H */
